doc/nrf/known_issues.rst

```diff
@@ -155,6 +155,30 @@ Thread
 
 .. rst-class:: v1-4-0
 
+KRKNWK-6848: Reduced throughput observed.
+  Performance testing for NCP sample shows 20 % decreased throughput with regards with vanilla OpenThread, passing from about 110 kbps to about 89 kbps.
+
+.. rst-class:: v1-4-0
+
+KRKNWK-7885: Throughput is lower when using CC310 nrf_security backend.
+  CC310 nrf_security backend is used by default for nRF52840 boards, but a 5-10 % decreased throughput has been observed when compared with nrf_oberon or vanilla mbedtls backends.
+
+  **Workaround:** Set the :option:`CONFIG_CC3XX_BACKEND` to ``n`` when building your samples.
```

Comment:
Lets add an info to this workaround that this will switch automatically to using Oberon software crypto backend (if I am not mistaken), so that a customer is aware he will not make use of hardware crypto acceleration.